scripts/update_teams.py

```diff
@@ -137,7 +137,10 @@ def create_team(org, name, description, repos):
 
 # Remove any teams which don't belong any more (because there is no longer a feedstock).
 for team_to_remove in set(teams.keys()) - set(packages_visited):
-    if team_to_remove in ['Core', 'conda-forge.github.io', 'all-members']:
+    if team_to_remove in ['Core',
+                          'conda-forge.github.io',
+                          'all-members',
+                          'conda-forge-webservices']:
```

Comment:
Line got too long so I wrapped it. Don't know if this is how you like to do your line wraps or not. Can change if needed.